tests/test_target_database.py

```diff
@@ -1,46 +1,52 @@
 """Tests for `mbed_targets`."""
 
-import unittest
+from unittest import TestCase
+
+import pytest
```

Comment:
I don't think this is needed any longer

Reply:
31fdc5e8dd760b77543315b2a32912061873f170